src/components/buttonCo.js

```diff
@@ -119,6 +119,38 @@ const useClasses = makeStyles(() => ({
       color: palette.common.white,
     },
   },
+  planePrimary: {
```

Comment:
Why did you create it manually and not in the override? 

![image](https://user-images.githubusercontent.com/13072856/83692170-19497a00-a5b9-11ea-9e89-efb09f983ccf.png)

Reply:
So I tried this one but it is a button without variant. The problem is when I try to send the variant empty or just dont send it, it simply applies the "outlined" variant and the button has a border. It is because of the default value that's being set on the ButtonCo component. Also some overrides like the disabled didn't applied. 

I would just leave it there, since those are already custom and I dont want to affect any other button by overriding the styles

Reply:
If you want to try, I defined the default outline applying this:
```jsx
  outlined: {
    ...outlinedPrimary,
  },
  outlinedPrimary,
```

changing the default **outline**